scripts/get-next-version.js

```diff
@@ -8,44 +8,95 @@ const bumpCb = require('conventional-recommended-bump')
 const { promisify } = require('util')
 
 const currentVersion = require('../package.json').version
-
+const { changeCatagories } = require('./semantic-commits/change-categories')
 const bump = promisify(bumpCb)
 const paths = ['packages', 'cli']
 
-let nextVersion
-
 const getNextVersionForPath = async (path) => {
   // allow the semantic next version to be overridden by environment
   if (process.env.NEXT_VERSION) {
     return process.env.NEXT_VERSION
   }
 
-  const { releaseType } = await bump({ preset: 'angular', path })
-
-  return semver.inc(currentVersion, releaseType || 'patch')
-}
+  let commits
+  const whatBump = (foundCommits) => {
+    // semantic version bump: 0 - major, 1 - minor, 2 - patch
+    let level = 2
+    let breakings = 0
+    let features = 0
+
+    commits = foundCommits
+    foundCommits.forEach((commit) => {
+      if (!commit.type || !changeCatagories[commit.type]) return
+
+      if (changeCatagories[commit.type].release === 'major') {
+        breakings += 1
```

Comment:
breakings could be a bool right?

Reply:
`breakings` is just counting the number of breaking changes for the reason below: `There is ${breakings} BREAKING CHANGE and ${features} features`. Just a nice-to-have insight into why the level is 0 or 1 or 2.